### ISSUE TITLE
Move to a layered design with separation of web routing, business logic and data access

### ISSUE THREAD

**Issue by dirrelito** [dirrelito/sprakpolisen-ML-use#4]
To simplify the code base for others to read, some more modularization of the code is appropriate. After implementation of #3  one should separate ML logic from web service functionality.

While already at it, data access should also be abstracted away to separate data service modules. The goal is to reach something like:

```
  HTTP Request       Response      Calls to external data services
       +                ^                    ^
       |                |                    |
       |                |                    |
       |                |                    |
       v                -                    |
                                             |
+--------------------------------------+     |
|                                      |     |
|  Routing layer via express.js        |     |
|                                      |     |
+--------------------------------------+     |
|                                      |     |
|  ML logic layer in "controllers"     |     |
|                                      |     |
+--------------------------------------+     |
|                                      |     |
|  Data access layer with files/http   +-----+
|                                      |
+--------------------------------------+

```

**Comment by dirrelito** [comment]
I've decided to implement it like:

```

  HTTP Request       Response      Calls to external data services
       +                ^                    ^
       |                |                    |
       |                |                    |
       |                |                    |
       v                -                    |
                                             |
+--------------------------------------+     |
|                                      |     |
|  ./app.js        |     |
|                                      |     |
+--------------------------------------+     |
|                                      |     |
|  ./actions     |     |
|                                      |     |
+--------------------------------------+     |
|                                      |     |
|  ./support   +-----+
|                                      |
+--------------------------------------+


```

This is somewhat inspired by the MVC and similar patterns. But I think the term "Controller" is strange in a restful setting, and we dont have any views either. So for now its a "Action-Support" pattern. :hankey:

**Comment by dirrelito** [comment]
Closed by PR #3
